x/inflation/keeper/epoch_mint_provisions.go

```diff
@@ -0,0 +1,36 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tharsis/evmos/x/inflation/types"
+)
+
+// GetEpochMintProvision gets the current EpochMintProvision
+func (k Keeper) GetEpochMintProvision(ctx sdk.Context) (sdk.Dec, bool) {
+	store := ctx.KVStore(k.storeKey)
+	bz := store.Get(types.KeyprefixEpochMintProvisions)
+	if len(bz) == 0 {
+		return sdk.ZeroDec(), false
+	}
+
+	var epochMintProvision sdk.Dec
+	err := epochMintProvision.Unmarshal(bz)
+	if err != nil {
+		panic(fmt.Errorf("unable to unmarshal epochMintProvision value %v", err))
```

Comment:
nit, use `%w` to wrap the error
```suggestion
		panic(fmt.Errorf("unable to unmarshal epochMintProvision value: %w", err))
```